scitos_mira/src/ScitosDrive.cpp

```diff
@@ -65,7 +66,7 @@ void ScitosDrive::initialize() {
 }
 
 void ScitosDrive::velocity_command_callback(const geometry_msgs::Twist::ConstPtr& msg) {
-  if (! barrier_status_.barrier_stopped ) {
+  if (! barrier_status_.barrier_stopped || ! emergency_stop_.data) {
```

Comment:
yes @cburbridge is right (it should only run when both are false) I'll fix it only makes me wonders how it worked